app/views/quizmaster/quizzes/show.html.haml

```diff
@@ -1,11 +1,15 @@
 .big-box
   %h1= @quiz.name
   %h3= @quiz.code
-  .activate_quiz
-    = form_tag quizmaster_path(@quiz), id: 'start_quiz', remote: true do
-      = hidden_field_tag 'welcome', true
-      = text_field_tag 'message', nil, placeholder: 'Welcome message', class: 'form-control code'
-      = submit_tag 'Start the Quiz', id: 'start_button', class: 'btn btn-primary btn-lg btn-block'
+  - if @quiz.is_started == false
+    .activate_quiz
```

Comment:
I'm not sure what you mean by your original statement either. It seems like the code stays the same but just moves around if I use `if true` or `unless`...